The problem is that if the ganeti package is installed, but this node wasn't added to the cluster yet, logrotate send an email like this every day:

Corrupt configuration file: Can't read ssconf file /var/lib/ganeti/ssconf_master_node: [Errno 2] No such file or directory: '/var/lib/ganeti/ssconf_master_node'
Aborting.

--- doc/examples/ganeti.logrotate.in.orig	2015-11-13 09:05:36.000000000 +0000
+++ doc/examples/ganeti.logrotate.in	2016-03-24 04:17:30.683219771 +0000
@@ -7,6 +7,6 @@
         delaycompress
         sharedscripts
         postrotate
-		@PKGLIBDIR@/daemon-util rotate-all-logs
+		service ganeti rotate --quiet
         endscript
 }
